src/modules/fancyzones/editor/FancyZonesEditor/Models/LayoutModel.cs

```diff
@@ -219,21 +220,42 @@ public static ObservableCollection<LayoutModel> LoadCustomModels()
                     var info = current.GetProperty(InfoJsonTag);
                     if (type.Equals(GridJsonTag))
                     {
+                        bool error = false;
+
                         int rows = info.GetProperty(RowsJsonTag).GetInt32();
                         int columns = info.GetProperty(ColumnsJsonTag).GetInt32();
 
+                        if (rows <= 0 || columns <= 0)

```

Comment:
In addition to this check `rows` and `columns` should be equal to "rows-percentage", "columns-percentage", "cell-child-map" sizes. Otherwise there could be a crash or inappropriate behavior.

Reply:
Done